src/App.js

```diff
@@ -5,11 +5,11 @@ import Form from './Form';
 
 function App() {
   return (
-    <div className="App">
+    <>
```

Comment:
We should probably add a div with a class of wrapper, either in here or in the form component itself.